### PR TITLE
9583 task: adds audio player icons

### PR DESCRIPTION
**Pull request by skibinska**
Relates https://github.com/wellcometrust/corporate/issues/9583

### This PR

- adds audio player icons: play, pause, volume and mute
- renames `IconActionPlay` icon to `IconActionPlayCircle` (is it okay to do that? or I should keep the play icon and new icon should be PlayWithoutCircle??)

Screenshot 2022-04-04 at 17 05 03

|   Added icons |   |   |
|---|---|---|
| Screenshot 2022-04-04 at 17 09 37|Screenshot 2022-04-04 at 17 16 26 | Screenshot 2022-04-04 at 17 09 25 |

### CODE REVIEW

**Comment by creido-welly** [comment]
@skibinska have these icons been tested in a light colour against a dark background?

Looking at the SVG I suspect some of the shapes aren't cut out correctly.

File: src/components/Icon/components/IconActionPlay.tsx [hunk 1]
```diff
@@ -1,12 +1,24 @@
 import React from 'react';
 
 const IconActionPlay = ({ viewBox = '0 0 24 24' }: { viewBox?: string }) => (
-  <svg viewBox={viewBox}>
-    <path
-      d="M12 0c6.627 0 12 5.373 12 12s-5.373 12-12 12S0 18.627 0 12 5.373 0 12 0zm0 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zM9 7.618c.155 0 .308.036.447.106l6.764 3.382a1 1 0 0 1 0 1.788l-6.764 3.382A1 1 0 0 1 8 15.382V8.618a1 1 0 0 1 1-1z"
-      fill="currentColor"
-      fillRule="evenodd"
-    />
+  <svg viewBox={viewBox} xmlnsXlink="http://www.w3.org/1999/xlink">
```
Comment (creido-welly) [inline src/components/Icon/components/IconActionPlay.tsx:4]: Sorry, to be a pain. Please could you put this one through svgomg or svg-batch-transforms. There is a lot that can be simplified.

**Review by creido-welly — APPROVED** [review]
Thanks for humouring me 😄

File: src/components/Icon/components/IconActionPlay.tsx (hunk 1 above)
Comment (skibinska) [reply src/components/Icon/components/IconActionPlay.tsx:4]: I used the svg-batch-transforms but not a lot was updated, [see diff](https://github.com/wellcometrust/design-system/pull/32/commits/a35795ab1b33f7c5f019ff44cf20ae34d213a383)

**Pull request merged**